torch/_torch_docs.py

```diff
@@ -3892,6 +3892,45 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, range=None, weight=None, density=False, out=None) -> (Tensor, Tensor)
+
+Computes the histogram of the values in a tensor.
+
+If :attr:`bins` is an int, it specifies the number of equal-width bins.
+:attr:`range` may be used to specify the lower and upper range of the bins.
+If no range is specified, the minimum and maximum elements of the input
```

Comment:
The order of these sentences seems logically reversed, since docs should explain the default behavior first, right?

"By default, the bins ... The :attr:\`range\` argument can be provided to specify a different range for the bins."

Maybe something like that?